packages/runtime/datastore-definitions/src/channel.ts

```diff
@@ -102,6 +102,12 @@ export interface IChannelStorageService {
      */
     read(path: string): Promise<string>;
 
+    /**
+     * Reads the object contained at the given path. Returns a utf8 string representation for the object.
+     */
+    readString(path: string): Promise<string>;
+
+    readBlob(path: string): Promise<IBlob>;
```

Comment:
Ideally it is consistent with storage interfaces and returns IsoBuffer.
Actual internal implantation can use helper functions that return IBlob, but I do not think there is a reason for public interface to have it